handlers/create.go

```diff
@@ -125,18 +129,24 @@ func logProgress(readerStdout io.Reader, readerStderr io.Reader, publishChan cha
 	}
 }
 
-func filterDockerMessage(msg string, uuid string, errString *string) string {
+func filterDockerMessage(msg string, machine *client.Machine, errString *string) string {
 	// Docker log messages come in the format: time=<t> level=<log-level> msg=<message>
 	// We just want to return <message> to cattle and only messages that do not contain the machine uuid
-	if strings.Contains(msg, uuid) {
+	if strings.Contains(msg, machine.ExternalId) || strings.Contains(msg, machine.Name) {
+		return ""
+	}
+
+	// The minimum string should be greater than 7 characters msg="."
+	match := RegExDockerMsg.FindString(msg)
+	if len(match) < 7 || !strings.HasPrefix(match, "msg=\"") {
```

Comment:
`RegExDockerMsg` is currently `"msg=.*"` so how would it ever not have this prefix?